<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bg
{background-color:qlineargradient(spread:pad, x1:0.04, y1:0.948864, x2:1, y2:0, stop:0 rgba(71, 82, 218, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QGroupBox" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLabel" name="vantaparent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;
</string>
     </property>
     <property name="text">
      <string>Wind's Apparent Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="vaparvant">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Wind's Apparent Speed = </string>
     </property>
    </widget>
    <widget class="QLabel" name="vn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Ship's Speed = </string>
     </property>
    </widget>
    <widget class="QLabel" name="da">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Ship's Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="valda">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvitnav">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvantapar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valdirvant">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLabel" name="vitvantadev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Real Wind Speed = </string>
     </property>
    </widget>
    <widget class="QLabel" name="vantadev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Real Wind Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="valvantadev">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvitadevvant">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="on">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;ON&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;ON&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>PLOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="derivalabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>360</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Bahnschrift Light&quot;;</string>
    </property>
    <property name="text">
     <string>Ship's adrift</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="input_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Plotting data</string>
    </property>
    <widget class="QLabel" name="vantaparent_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;
</string>
     </property>
     <property name="text">
      <string>Wind's Apparent Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="vn_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Ship's Speed = </string>
     </property>
    </widget>
    <widget class="QLabel" name="da_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Ship's Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="valda_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>25</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvitnav_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvantapar_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valdirvant_2">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="vantadev_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Real Wind Heading = </string>
     </property>
    </widget>
    <widget class="QLabel" name="valvantadev_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valvitadevvant_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="vaparvant_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Wind's Apparent Speed = </string>
     </property>
    </widget>
    <widget class="QLabel" name="vitvantadev_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Ship's Heading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>Real Wind Speed = </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
